### DISCUSSION TITLE
Ignore a repository

### DISCUSSION THREAD

**Discussion by arunsathiya** [anuraghazra/github-readme-stats#3597] [Q&A] [answered]
Is there a way to ignore a repository from counting towards the stats? I have searched a bit on existing discussions and issues but I am not sure I spot that functionality yet.

**Comment by qwerty541** [comment, marked as answer]
Hey, @arunsathiya! You can use `exclude_repo` query option to ignore repository from counting towards the stats. This option is available for both stats and top languages cards. Here is an example:

```html
<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&exclude_repo=github-readme-stats" />
```

<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&exclude_repo=github-readme-stats" />

**Reply by arunsathiya** [reply]
That's awesome. Thank you, @qwerty541!

**Reply by qwerty541** [reply]
I am glad to help! Feel free to contact me if you need any further assistance.

**Reply by arunsathiya** [reply]
@qwerty541 Unfortunately, I am having some trouble getting it working. I added the query param about 20 hours ago, but I don't see a change yet:

https://github.com/arunsathiya/arunsathiya/blob/main/README.md?plain=1

**Reply by arunsathiya** [reply]
Please ignore my last comment. I seem to have a few more notebook repositories which can add to the top languages count. I'll exclude them too.

**Reply by qwerty541** [reply]
@arunsathiya I just tried two examples below with your top languages card. After addition of `exclude_repo` query option it shows different percentages.

About stats card, as i understood, `exclude_repo` option affect only on count of received stars. Here is the code: https://github.com/anuraghazra/github-readme-stats/blob/master/src/fetchers/stats-fetcher.js#L309-L315

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=arunsathiya&show_icons=true" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=arunsathiya&show_icons=true" />

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=arunsathiya&exclude_repo=spotify-with-langchain&show_icons=true" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=arunsathiya&exclude_repo=spotify-with-langchain&show_icons=true" />
